mbed_app.json

```diff
@@ -27,6 +27,12 @@
         "AIN_4": "A4",
         "AIN_5": "A5",
         "AOUT":  "A5",
+        "AOUT_0": "A0",

```

Comment:
Are really all Ax pins capable of DAC?

AOUT should be removed to be consistent ? Pins Ax start from 0 for instance